src/applications/disability-benefits/686c-674/tests/e2e/00-spouse-child-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,197 @@
+  client.click('button[id="4-continueButton"]');
+
+  // spouse information
+  E2eHelpers.expectLocation(client, '/add-spouse');
+  client.waitForElementVisible(
+    '#root_spouseFullName_first-label',
+    Timeouts.normal,
+  );
+  client.axeCheck('.main');
+  TestHelpers.fillSpousePersonalInformation(client, testData.data);
+  client.click('button[id="4-continueButton"]');
+
+  // current marriage information
+  E2eHelpers.expectLocation(client, '/current-marriage-information');
+  client.waitForElementVisible('#root_dateOfMarriage-label', Timeouts.normal);
+  client.axeCheck('.main');
+  TestHelpers.fillCurrentMarriageInformation(client, testData.data);
+  client.click('button[id="4-continueButton"]');
+
+  // current spouse address
+  E2eHelpers.expectLocation(client, '/current-marriage-address');
+  client.waitForElementVisible(
+    '#root_spouseDoesLiveWithVeteran-label',
+    Timeouts.normal,
+  );
+  client.axeCheck('.main');
+  client.pause(Timeouts.normal);
+  TestHelpers.fillSpouseAddressInformation(client, testData.data, false);
+  client.click('button[id="4-continueButton"]');
+
+  // current spouse marriage history
+  E2eHelpers.expectLocation(client, '/current-spouse-marriage-history');
+  client.waitForElementVisible(
+    '#root_spouseWasMarriedBefore-label',
+    Timeouts.normal,
+  );
+  client.axeCheck('.main');
+  client.pause(Timeouts.normal);
+  TestHelpers.fillSpouseMarriageHistory(client, testData.data, true);
+  client.click('button[id="4-continueButton"]');
+
+  // spouse marriage history details
+  E2eHelpers.expectLocation(client, '/current-spouse-marriage-history/0');
+  client.waitForElementVisible(
+    '#root_marriageStartDate-label',
+    Timeouts.normal,
+  );
+  client.axeCheck('.main');
+  TestHelpers.fillSpouseMarriageHistoryDetails(client, testData.data);
+  client.click('button[id="4-continueButton"]');
+
+  // veteran marriage history
+  E2eHelpers.expectLocation(client, '/veteran-marriage-history');
+  client.waitForElementVisible(
+    '#root_veteranWasMarriedBefore-label',
+    Timeouts.normal,
+  );
+  client.axeCheck('.main');
+  client.pause(Timeouts.slow);
+  TestHelpers.fillVeteranMarriageHistory(client, testData.data, true);
+  client.click('button[id="4-continueButton"]');
+
+  // veteran marriage history details
+  E2eHelpers.expectLocation(client, '/veteran-marriage-history/0');
+  client.waitForElementVisible(
+    '#root_marriageStartDate-label',
+    Timeouts.normal,
+  );
+  TestHelpers.fillVeteranMarriageHistoryDetails(client, testData.data);
+  client.click('button[id="4-continueButton"]');
+
+  // marriage additional evidence
+  E2eHelpers.expectLocation(client, '/add-spouse-evidence');
+  client.waitForElementVisible(
+    '#root_supportingDocuments_add_label',
+    Timeouts.normal,
+  );
+  client.pause(Timeouts.normal);
+  client.click(
+    '.row.form-progress-buttons.schemaform-buttons div.small-6.medium-5.end.columns button.usa-button-primary',
+  );
```

Comment:
This block just navigates through document upload without attempting to upload. I noticed there was another e2e test on vets-website that tried document upload without much success either (hca I think). It's definitely something to come back to, but all document upload is optional in this form and I didn't want to block this pull request on this one issue.